src/NuGet.Clients/NuGet.VisualStudio.Implementation/Extensibility/VsProjectJsonToPackageReferenceMigrateResult.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.PackageManagement.VisualStudio;
+
+namespace NuGet.VisualStudio
+{
+    internal class VsProjectJsonToPackageReferenceMigrateResult : IVsProjectJsonToPackageReferenceMigrateResult
+    {
+        public bool IsSuccess { get; }
+
+        public string ErrorMessage { get; }
+
+        public VsProjectJsonToPackageReferenceMigrateResult(ProjectJsonToPackageRefMigrateResult result)
+        {
+            IsSuccess = result.IsSuccess;
+        }
+
+        public VsProjectJsonToPackageReferenceMigrateResult(string errorMessage)
+        {
+            IsSuccess = false;
+            ErrorMessage = errorMessage;
```

Comment:
should this provide a generic error if this is null or empty? It looks like an odd exception with no message could creep in from other code